### PR TITLE
Webpage コンセントについて

### PR DESCRIPTION
**Pull request by marcchataigner**
Fixes # .

Changes proposed in this pull request:
-
-
-

@consento-org/maintainer

### CODE REVIEW

**Comment by marcchataigner** [comment]
Oops, I didn't write the detail of the PR ; it is to add the Japanese page detailing Consento in nihongo + a link to the page in the footer.

File: docusaurus.config.js [hunk 1]
```diff
@@ -54,6 +54,7 @@ module.exports = {
           items: [
             { label: 'Team', to: 'team', },
             { label: 'About', to: 'about', },
+            { label: 'コンセントについて', to: '/consento-JP', },
```
Comment (dkastl) [inline docusaurus.config.js]: why not name it `about-jp` ... Docusaurus should actually support localization as well.

**Review by dkastl — COMMENTED** [review]
So this would be using i18n by Docusaurus: https://docusaurus.io/docs/i18n/introduction

File: docusaurus.config.js (hunk 1 above)
Comment (marcchataigner) [reply docusaurus.config.js]: That would be a good idea, I didn't know about it...  I should have asked!  :-)

Comment (marcchataigner) [reply docusaurus.config.js]: so the naming would be 'about-JP' and 'about-EN', right?

Comment (dkastl) [reply docusaurus.config.js]: I would only specific languages use a language identifier but keep the default language `about` in this case.

**Review by martinheidegger — DISMISSED** [review]
Please use yourself, or @neo-wsbi, as author of this, as it is not a translation of things I wrote.

File: docusaurus.config.js (hunk 1 above)
Comment (marcchataigner) [reply docusaurus.config.js]: Hi Daniel, let me know if the i18 worked or if it is better (easier for now) to just implement this PR

**Comment by dkastl** [comment]
@marcchataigner , I made a few adjustments:
- updated authors
- named the page `jp` as this is a Japanese landing page for now (and no other localized content planned as fas as I understand)
- move support link in the footer.

**Comment by dkastl** [comment]
If we want to change to `i18n` this can be done in a new pull request, but for now it's fine as it is, I would say.
In case there will be a localized version someday `/jp` will probably be the index page.

File: src/pages/jp.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+Consentoでは、エスクローシステムが作成され、異なるパーティに分割されます。パーティの数は調整可能です。データは異なるデバイス間で同期が保たれ、データはいつでも検証することができます。
+
+---
+
+## ソリューション
+Consentoのアプリは、あなたの組織にカスタムメイドのHuman-Factor Authenticationを構築することを可能にします。
+ 
+<figure className="kg-card kg-image-card kg-card-hascaption">
+  <img src="/img/tanja/cc-by-nc-sa/icons/passwordless.svg" style={{ float: 'left', width: '20%' }} />
```
Comment (dkastl) [inline src/pages/jp.md:52]: @marcchataigner , there is no file with this name, so the image does not show up.

File: src/pages/jp.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+  <img src="/img/tanja/cc-by-nc-sa/icons/icon_lock.svg" style={{ float: 'left', width: '20%' }} />
+<figcaption><strong>クラウドフリー</strong>なので、中央管理者を必要とせず、認証プロセスはユーザーだけのものとなります。あなたのデータ管理はあなたの財産です。</figcaption>
+</figure>
+<br/>
+
+---
+
+<br/>
+<figure className="kg-card kg-image-card kg-card-hascaption">
+  <img src="/img/tanja/cc-by-nc-sa/icons/collaborative.svg" style={{ float: 'left', width: '20%' }} />
```
Comment (dkastl) [inline src/pages/jp.md:80]: @marcchataigner , there is no file with this name, so the image does not show up.

File: src/pages/jp.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+<figcaption><strong>相互の応答性</strong>Consentoは、組織内やサプライヤー、顧客に配布される部分的な鍵によって、認証をより安全にします。一人のユーザーがシステム全体の故障の原因になることはありません。</figcaption>
+</figure>
+<br/>
+
+---
+
+<br/>
+<figure className="kg-card kg-image-card kg-card-hascaption">
+  <img src="/img/tanja/cc-by-nc-sa/icons/icon_lock.svg" style={{ float: 'left', width: '20%' }} />
```
Comment (dkastl) [inline src/pages/jp.md]: This is the same icon as two sections up.

**Review by dkastl — CHANGES_REQUESTED** [review]
@marcchataigner , before merging this pull request, the images should be right, I think.

File: src/pages/jp.md (hunk 4 above)
Comment (marcchataigner) [reply src/pages/jp.md]: I update the file, and replaced the repeated image

File: src/pages/jp.md (hunk 3 above)
Comment (marcchataigner) [reply src/pages/jp.md:80]: on my side, it does show. Did you pull the last version? 

Screen Shot 2021-05-11 at 15 19 54

File: src/pages/jp.md (hunk 2 above)
Comment (marcchataigner) [reply src/pages/jp.md:52]: on my side, it does show. Did you pull the last version? 

Screen Shot 2021-05-11 at 15 19 54

Comment (dkastl) [reply src/pages/jp.md:52]: In my case the image files are not there, and I did `git fetch` and `git pull`

Comment (dkastl) [reply src/pages/jp.md:52]: The path says `/img/tanja/cc-by-nc-sa/icons/passwordless.svg`, but there is no such file in the folder:
https://github.com/consento-org/consento-org.github.io/tree/webpage-%E3%82%B3%E3%83%B3%E3%82%BB%E3%83%B3%E3%83%88%E3%81%AB%E3%81%A4%E3%81%84%E3%81%A6/static/img/tanja/cc-by-nc-sa/icons

Comment (marcchataigner) [reply src/pages/jp.md:52]: I added manually the missing PNG and SVG. It should work now (ideally). 

I wonder why these files where in my local repo but not in that branch online. Perhaps my SourceTree sinchronsation issue is still not resolved...

Comment (neo-wsbi) [reply src/pages/jp.md:52]: The Japanese descriptions sound bit weird. How can I adjust the translations on the solution page?

File: src/pages/jp.md (hunk 3 above)
Comment (dkastl) [reply src/pages/jp.md:80]: Now images are there!

File: docusaurus.config.js (hunk 1 above)
Comment (dkastl) [reply docusaurus.config.js]: Changes this to `/jp`

**Review by dkastl — APPROVED** [review]

File: src/pages/jp.md (hunk 2 above)
Comment (marcchataigner) [reply src/pages/jp.md:52]: I guess, here is the file: https://github.com/consento-org/consento-org.github.io/edit/webpage-コンセントについて/src/pages/jp.md

**Comment by martinheidegger** [comment]
Rebased.

**Review by martinheidegger — APPROVED** [review]

**Pull request merged**